BUILDING.md

```diff
@@ -77,12 +77,22 @@ binary artifacts are also made available to your Python environment.
 ## Running tests
 
 Testing is disabled by default and must be enabled by setting the
-`OPENASSETIO_ENABLE_TESTS` CMake variable. Assuming that the working
-directory is the repository root, we can configure the CMake build as
-follows
+`OPENASSETIO_ENABLE_TESTS` CMake variable.
+
+In addition, test fixtures can be configured such that a Python
+environment is created during the test run by setting
+`OPENASSETIO_ENABLE_PYTHON_TEST_VENV`. If this is `OFF` (default) then
```

Comment:
[question] Im struggling to remember why we said `OFF` by default, rather than `ON` (I know we definitely had good reason at the time!). Once we've de-coupled the venv and `dist` might it make sense to be on by default?

Reply:
Wasn't it that users are most likely to want to run the test suite against their Python environment, to test integration?

Reply:
🤔 Probably was wasn't it. I guess the Q is, if the Quickstart instructions always enable it - should we just enable it?

I could see the argument for it being off if the default instructions for the current early adopters left it so?

Reply:
This is enabled by default now. Part of [this commit](https://github.com/feltech/OpenAssetIO/commit/c66b14d9ac9e036690515e4e813c32cd6ce2f23b)